vespalib/src/vespa/vespalib/util/compress.h

```diff
@@ -82,6 +105,9 @@ class Integer {
         }
         return numbytes;
     }
+private:
+    [[ noreturn ]] static void too_big(int64_t n);
+    [[ noreturn ]] static void too_big(uint64_t n);
```

Comment:
Consider renaming to `throw_too_big`, or similar. Just `too_big` could at a glance be easy to confuse with a function that just _checks_ if a number is too big.

Reply:
Good point.